### ISSUE TITLE
terminal text remains colored after exit

### ISSUE THREAD

**Issue by rathann** [stolk/energygraph#1]
Terminal text remains coloured with the graph colour after exiting energygraph.

**Steps to reproduce:**
1. `sudo ./energygraph`
2. press `q`

**Expected results:**
Shell prompt text is the same colour as before (e.g. light grey)

**Actual results:**
Shell prompt text is the colour of the graph.

**Additional info:**
I'm using MATE desktop and MATE terminal (`mate-terminal`) on Fedora 36 if that matters.

Also, very cool piece of code, thanks for writing and publishing it!

**Comment by stolk** [comment]
I can't repro here: The shell text has the colour as expected.

But I do have a lot of black background in the terminal that goes away as you type.
Are you referring to the black background?

![image](https://user-images.githubusercontent.com/455280/192316863-a7a32a56-2ac8-46ba-bb6f-6472767f655a.png)

**Comment by rathann** [comment]
No, it's the text colour that's affected. However, I can't reproduce it on another machine with the same OS and desktop. I'll post a screenshot when I can access the first machine.

**Comment by stolk** [comment]
Can you try latest version?

**Comment by FFDA** [comment]
I have the same issue. And made a screenshots. If, after using the energygraph, I do a reset and go down to the end of the screen with <ENTER> - terminal distortion disappears.

I'm running openSUSE 15.3, KDE 5.76.0 / Plasma 5.18.6

![colors](https://user-images.githubusercontent.com/8406967/193236222-1c7034f1-2aba-4510-b694-4498a19d6703.png)

**Comment by stolk** [comment]
> I have the same issue. And made a screenshots. If, after using the energygraph, I do a reset and go down to the end of the screen with - terminal distortion disappears.

Was that with the latest changes? Does it happen with the version that is currently in the repo?
Thanks!

**Comment by FFDA** [comment]
I just start it off by writing that I do not think of this as a really big bug. One can simply close the terminal and be done with it. And I made the screenshots because the other person didn't follow up.

Now when I exit using q the colors are reset to default ones. If I do CTRL+C it still acts as in screenshots.
